packages/react-examples/src/react/ContextualMenu/ContextualMenu.Header.Example.tsx

```diff
@@ -44,5 +44,13 @@ export const ContextualMenuHeaderExample: React.FunctionComponent = () => {
     ],
   }));
 
-  return <DefaultButton text="Click for ContextualMenu" menuProps={menuProps} />;
+  return (
+    <>
+      <p>
+        Note: this example demonstrates how to use the Header menu item type as a standalone menu item. For semantically
+        grouped options, refer to the "Contextual Menu with section headers" example.
```

Comment:
Is there a well known URL so a link could be here.

Reply:
It's an example on the same page, and unfortunately we don't have id refs to link to 😓